addons/nuxeo-platform-document-routing/nuxeo-routing-rest-api/src/test/java/org/nuxeo/ecm/restapi/server/WorkflowEndpointTest.java

```diff
@@ -1239,11 +1240,17 @@ public void testTaskWorkflowInfo() throws IOException {
             JsonNode initiatorNode = taskNode.get("workflowInitiator");
             assertEquals("user", initiatorNode.get("entity-type").textValue());
             assertEquals("Administrator", initiatorNode.get("id").textValue());
-            assertTrue(initiatorNode.get("isAdministrator").booleanValue());
             JsonNode properties = initiatorNode.get("properties");
             ArrayNode groups = (ArrayNode) properties.get("groups");
-            assertEquals(1, groups.size());
-            assertEquals("administrators", groups.get(0).textValue());
+            if (initiatorNode.get("isPartial") != null && initiatorNode.get("isPartial").booleanValue()) {
```

Comment:
This is not in sync with the master PR that was merged (local variable used here). I see other changes. Could you compare master and 10.10 commits to make sure they are as close as possible (within the constraints of 10.10)?

Reply:
It should be only the changes needed for 10.10 now.